app/views/landing_page/_meta.erb

```diff
@@ -61,3 +61,6 @@
   <meta name="twitter:description" content="<%= p["description"]["value"] %>" />
 <% end %>
 
+<% if p["google_site_verification"] %>
+  <meta name="google_site_verification" content="<%= p["google_site_verification"]["value"] %>" />
```

Comment:
@Gnito Since I am not very good at code I just "copied" it and modified from the line above for copyright that is : 
`<% if p["copyright"] %>
  <meta name="copyright" content="<%= p["copyright"]["value"] %>" />
<% end %>`
I tried this code in preview and it works... but I don't know if it will work in prod :)